moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -532,9 +539,24 @@ KinematicState Servo::getNextJointState(const ServoInput& command)
     {
       RCLCPP_DEBUG_STREAM(logger_, "Joint velocity limit scaling applied by a factor of " << joint_limit_scale);
     }
-
     target_state.velocities *= joint_limit_scale;
 
+    // Scale down the acceleration based on joint acceleration limit if applicable.
+    // TODO move to function: ApplyAccelerationLimit(...)
```

Comment:
Did you mean to do this TODO for this PR? I'm fine either way, but I'd say either implement this in this PR or remove the comment.

Reply:
I'd rather get rid of the acceleration limiting here completely. IMO it belongs as an online_signal_smoothing plugin to replace the ButterworthFilter. It should be a drop-in replacement with no code changes required in the Servo pkg.

Reply:
This is what @pac48 had in previous versions of this PR, right?